### PR TITLE
Project setup section added to readme

### PR DESCRIPTION
**Pull request by ivacf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,3 +55,17 @@ The project uses a checkstyle plugin to make sure that the code follows our [And
...
+To quickly start a new project from this boilerplate follow the next steps:
+
+* Download this [repository as a zip](https://github.com/ribot/android-boilerplate/archive/master.zip).
+* Change the package name. 
+  * Rename packages in main, androidTest and test using Android Studio.
+  * In `app/build.gradle` file, `packageName` and `testInstrumentationRunner`.
+  * In `src/main/AndroidManifest.xml` and `src/debug/AndroidManifest.xml`.
+* Optionally, in `app/build.gradle` add the signing config to enable release versions.
+* Create a new git respository, [see GitHub tutorial](https://help.github.com/articles/adding-an-existing-project-to-github-using-the-command-line/).
+* Replace the test code with your app code following the same architecture.
```
Comment (jsonfry) [inline README.md]: What do you mean by `test code` ?

Comment (ivacf) [inline README.md]: probably `example code` makes more sense

Comment (jsonfry) [inline README.md]: yeah :)

**Comment by ivacf** [comment]
@jsonfry updated

**Pull request merged**